src/components/Tickets/DetailsModal/Notes.js

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
-import { fetchTicketNotes } from '../../../helpers/cw';
+import { ref } from '../../../config/constants';
 
 const displayNotes = (ticketNumber, setIsLoading, setNotes) => {
   setIsLoading(true);
-  fetchTicketNotes(ticketNumber).then(results => {
-    const notes = results.map(note => ({
-      createdBy: note.member.name,
-      dateCreated: (new Date(note.dateCreated)).toLocaleDateString() + ' ' + (new Date(note.dateCreated)).toLocaleTimeString(),
-      id: note.id,
-      text: note.text,
-    }));
-
-    setIsLoading(false);
-    setNotes(notes);
+  const ticketNotes = ref.child(`notes/${ticketNumber}`);
+  ticketNotes.on('value', snapshot => {
```

Comment:
You probably want `once` here.  Or weird things might happen when someone has opened multiple tickets and someone else (or the cron) updates ticket data.

https://firebase.google.com/docs/reference/node/firebase.database.Reference#on

Reply:
Alternatively, could `useEffect()` and call `ticketNotes.off(returnValueOfOn);` in the forget function.  Then it would live update if the ticket notes were updated while they were viewing it.  Probably not needed.